GPU/Common/VertexShaderGenerator.cpp

```diff
@@ -474,10 +474,11 @@ bool GenerateVertexShader(const VShaderID &id, char *buffer, const ShaderLanguag
 			*uniformMask |= DIRTY_PROJTHROUGHMATRIX;
 		} else if (useHWTransform) {
 			if (IsVRBuild() && IsMultiviewSupported()) {
-				WRITE(p, "layout(shared) uniform ProjectionMatrix { uniform mat4 u_proj[2]; };\n");
+				WRITE(p, "layout(shared) uniform ProjectionMatrix { uniform mat4 u_proj_lens[2]; };\n");
 			} else {
-				WRITE(p, "uniform mat4 u_proj;\n");
+				WRITE(p, "uniform mat4 u_proj_lens;\n");
```

Comment:
Is this intentional?  Is this meant to be `} else if (IsVRBuild()) {`?

-[Unknown]